app/controllers/blackouts_controller.rb

```diff
@@ -93,10 +93,30 @@ def create # rubocop:disable MethodLength
   end
 
   def update
+    p = blackout_params
+    p[:created_by] = current_user.id 
```

Comment:
We shouldn't be updating `created_by` -- we want the record of whoever originally created the blackout